packages/atdgen-windows.1.10.0/files/atdgen.install

```diff
@@ -0,0 +1,6 @@
+bin: [
+  "src/atdgen.run" {"atdgen.run"}
+  "src/atdgen" {"atdgen"}
+  "atdgen-cppo/atdgen-cppo"
+  "atdgen-cppo/cppo-json"
+]
```

Comment:
Ditto here--if you install these at all, install them into the cross-root, or they will clobber the files installed by `atdgen` proper in the prefix, and likely breaking your build scripts.